build

```diff
@@ -73,6 +73,10 @@ while true; do
             PLATFORM="$2"
             shift 2
             ;;
+        --args)
+            ARGS="$2"
```

Comment:
Could we instead use rest arguments (everything after `--`)? E.g.:

`prebuildify-cross --platform=linux --arch=arm --arm-version=7 -- -t 8.14.0 --napi --strip`

Reply:
I don't mind at all.